src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -1178,9 +1178,9 @@ void testSaWithLoadContingency() {
 
         // post-contingency tests
         PostContingencyResult l4ContingencyResult = getPostContingencyResult(result, "l4");
```

Comment:
For me, there is no bug in main branch. On this case, I have run a power flow disconnecting l4 and the flows on branches are the actual ones and not what you have found. 
When we loose a load on a bus, we change 2 things :
- The bus `loadTargetP` of course ;
- We also change the variable part of loads connected to the bus, removing the part that has been lost. It will change the participating factor through method `getParticipationFactor`.
Why do you think there is a mistake?